cli/src/ui/UI.tsx

```diff
@@ -201,6 +202,11 @@ export const UI: React.FC<UIAppProps> = ({ options, onExit }) => {
 		}
 	}, [configValidation])
 
+	// If JSON mode is enabled, use JSON renderer instead of UI components
+	if (options.json && options.ci) {
```

Comment:
we 
```suggestion
	if (options.json) {
```

we already guarantee this right?